styles.css

```diff
@@ -342,7 +383,7 @@ body {
   padding: 0rem 2rem 0rem 2rem
```

Comment:
btw this can use shorthand and for `0` value, the unit doesnt need to be specific

so the shorthand would look like this: `padding: 0 2rem` . When two value is passed to the padding property, the top & bottom padding will take the first value (0 in this case) and the left & right padding will take second value (2rem)